docs/DeveloperGuide.md

```diff
@@ -138,7 +138,7 @@ Similar analogues exist for task and lesson data. The class diagram is similar a
 The above diagram only shows the UserPrefs and StudentBook Storage in full. TaskBook Storage is done similarly to StudentBook Storage. The only difference is the name of the classes (`JsonTaskBookStorage` instead of `JsonStudentBookStorage`, `JsonAdaptedTask` instead of `JsonAdaptedStudent` etc.)
```

Comment:
```suggestion

The above diagram only shows the UserPrefs and StudentBook Storage in full. TaskBook Storage is designed similarly to StudentBook Storage. The only difference is the name of the classes (`JsonTaskBookStorage` instead of `JsonStudentBookStorage`, `JsonAdaptedTask` instead of `JsonAdaptedStudent` etc.)
```